website/landing/components/Intro/Intro.tsx

```diff
@@ -0,0 +1,207 @@
+import { content } from "../../content/intro-section";
+import { Box, CodeBlock, List, Text, TextHighlight } from "../common";
+
+const ContentLandmark = () => {
+  return (
+    <Box
+      css={{
+        alignItems: "center",
+        background: "white",
+        color: "black",
+        display: "flex",
+        height: "595px",
+        justifyContent: "center",
+        width: "343px",
+
+        "@bp1": {
+          width: "376px",
+        },
+
+        "@bp2": {
+          height: "497.5px",
```

Comment:
Can we round these values? I don't think it would work on low pixel density devices

Reply:
@danilowoz sure! it's markup so when updating with the final content the styling should be refined.